xiaqianZhang/assignments/quiz/q1.py

```diff
@@ -0,0 +1,43 @@
+#Problem Statement
+#Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+'''
+Input: array of positive numbers and given with a positive integer k
+Output: return the max sum of continuous subarray with size of k
+
+edge case: if the array is empty, return 0
+
+initialize the startIndex = 0, currentSum = 0, maxSum = 0
+loop through each index in the given array:
+    append the value of the each index into the currentSum
+    if each index is greater than the size of k-1 (that means the subarray is outside of the boundary, we want to continue move towards the next subarray)
+        update the maxSum
+        currentSum decrease the value by the startIndex, since we want to continue looping to next subarray without repeat calculation
+        increment the startIndex by 1
+return maxSum
+'''
```

Comment:
Very nice overview! Don't forget to include the time and space complexity of your approach as well.